### PR TITLE
Make gaia-progress be WAI-ARIA progressbar

### PR DESCRIPTION
**Pull request by takenspc**
Spec:
- http://w3c.github.io/aria/aria/aria.html#progressbar

### CODE REVIEW

**Comment by takenspc** [comment]
My changes are same as the previous PR (https://github.com/gaia-components/gaia-progress/pull/1).
The difference is whether a context line has ";" or not.

File: gaia-progress.js [hunk 1]
```diff
@@ -19,6 +19,9 @@ module.exports = component.register('gaia-progress', {
       inner: this.shadowRoot.querySelector('.inner'),
       bar: this.shadowRoot.querySelector('.bar'),
     };
+    this.els.inner.setAttribute("role", "progressbar");
```
Comment (wilsonpage) [inline gaia-progress.js]: NIT: Extra line-break above this line

File: gaia-progress.js [hunk 2]
```diff
@@ -38,6 +41,9 @@ module.exports = component.register('gaia-progress', {
           var duration = (delta / 100) * this.fillTime;
           this.els.bar.style.transform = `translateX(${value}%)`;
           this.els.bar.style.transitionDuration = duration + 'ms';
+          this.els.inner.setAttribute("aria-valuenow", value);
+        } else {
+          this.els.inner.removeAttribute("aria-valuenow");
```
Comment (wilsonpage) [inline gaia-progress.js]: Could you convert all `"` to `'`, thanks :)

**Comment by takenspc** [comment]
Thank you for your review.

I addressed coding style issues.

**Comment by wilsonpage** [comment]
Perfect thanks :)

Can you 'squash'\* your commits into one and I'll press the merge button.

*You'll need to use an 'interactive rebase' something like `$ git rebase -i upstream/master`. If you are unfamiliar with this, there are lots of tutorials online.

**Comment by takenspc** [comment]
Thank you for your review. I squashed my commits.

**Pull request merged**